cot-macros/src/lib.rs

```diff
@@ -195,32 +195,7 @@ pub(crate) fn cot_ident() -> proc_macro2::TokenStream {
         }
     }
 }
-/// A derive macro that automatically implements the [`FromRequestParts`] trait
-/// for structs.
-///
-/// This macro generates code to extract each field of the struct from HTTP
-/// request parts, making it easy to create composite extractors that combine
-/// multiple data sources from an incoming request.
-///
-/// The macro works by calling [`FromRequestParts::from_request_parts`] on each
-/// field's type, allowing you to compose extractors seamlessly. All fields must
-/// implement the [`FromRequestParts`] trait for the derivation to work.
-///
-/// # Examples
-///
-/// ## Named Fields
-///
-/// ```no_run
-/// use cot::request::extractors::{Path, StaticFiles, UrlQuery};
-/// use cot::router::Urls;
-/// use cot_macros::FromRequestParts;
-///
-/// #[derive(Debug, FromRequestParts)]
-/// pub struct BaseContext {
-///     urls: Urls,
-///     static_files: StaticFiles,
-/// }
-/// ```
+
 #[proc_macro_derive(FromRequestParts)]
```

Comment:
The doc is already here, no need to duplicate it:

https://github.com/cot-rs/cot/blob/c0bb892dd33bcda5b71ba0b051c54631b3a2f139/cot/src/request/extractors.rs#L540-L573